### PR TITLE
Shorten big types labels

### PR DESCRIPTION
**Pull request by mslabek**

### CODE REVIEW

File: designer/client/src/stylesheets/processes.styl [hunk 1]
```diff
@@ -21,3 +21,9 @@
 .validation-labels
     display inline-grid
     max-width fit-content
+
+.line-cut
```
Comment (JulianWielga) [inline designer/client/src/stylesheets/processes.styl:25]: we don't want new stylus things
we want to get rid of stylus

File: designer/client/src/components/graph/node-modal/FieldLabel.tsx [hunk 2]
```diff
@@ -24,11 +24,21 @@ export function FieldLabel({
     const processDefinitionData = useSelector(getProcessDefinitionData);
     const params = getNodeParams(processDefinitionData, nodeId);
     const parameter = findParamDefinitionByName(parameterDefinitions, paramName);
-    const label = params?.[paramName]?.label ?? paramName;
+    const label = (params?.[paramName]?.label ?? paramName) + ":";
```
Comment (JulianWielga) [inline designer/client/src/components/graph/node-modal/FieldLabel.tsx:27]: the colon is not part of the label

File: designer/client/src/components/graph/node-modal/FieldLabel.tsx [hunk 3]
```diff
@@ -24,11 +24,21 @@ export function FieldLabel({
...
     return (
         <div className="node-label" title={paramName}>
-            {label}:{parameter ? <div className="labelFooter">{ProcessUtils.humanReadableType(parameter.typ)}</div> : null}
+            {label}
+            {parameter && <LabelFooter parameterType={parameter.typ} />}
+        </div>
+    );
+}
+
+function LabelFooter({ parameterType }: { parameterType: TypingResult }): JSX.Element {
```
Comment (JulianWielga) [inline designer/client/src/components/graph/node-modal/FieldLabel.tsx:37]: i see no value in this new component

File: designer/client/src/components/modals/ValidationLabels.tsx [hunk 4]
```diff
@@ -5,20 +5,18 @@ import { Validator, withoutDuplications } from "../graph/node-modal/editors/Vali
...
     validators: Array<Validator>;
     values: Array<string>;
-    additionalClassName?: string;
     validationLabelInfo?: string;
-    processNameValidationError?: string;
 };
 
 export default function ValidationLabels(props: Props) {
-    type ValidationErrors = {
+    type ValidationError = {
```
Comment (JulianWielga) [inline designer/client/src/components/modals/ValidationLabels.tsx:12]: also could/should be outsied of ValidationLabels

File: designer/client/src/components/modals/ValidationLabels.tsx [hunk 5]
```diff
@@ -29,22 +27,42 @@ export default function ValidationLabels(props: Props) {
 
     const renderErrorLabels = () =>
         validationErrors.map((validationError, ix) => (
-            <span key={ix} className="validation-label-error" title={validationError.description}>
-                {validationError.message}
-            </span>
+            // we don't pass description as tooltip message - we pass the entire non-line-limited string until we make
+            // changes on the backend
+            <LimitedValidationLabel key={ix} message={validationError.message} tooltipMessage={validationError.message} type={"ERROR"} />
```
Comment (JulianWielga) [inline designer/client/src/components/modals/ValidationLabels.tsx:32]: index is not the best key, especially here

File: designer/client/src/components/modals/ValidationLabels.tsx [hunk 6]
```diff
@@ -29,22 +27,42 @@ export default function ValidationLabels(props: Props) {
...
+
+type ValidationLabelProps = {
+    message: string;
+    tooltipMessage: string;
+    type: ValidationLabelType;
+};
+
+type ValidationLabelType = "INFO" | "ERROR";
+
+const labelTypeMap = {
```
Comment (JulianWielga) [inline designer/client/src/components/modals/ValidationLabels.tsx:56]: object keys are not connected to ValidationLabelType

File: designer/client/src/components/modals/ValidationLabels.tsx [hunk 7]
```diff
@@ -29,22 +27,42 @@ export default function ValidationLabels(props: Props) {
...
+    INFO: "validation-label-info",
+    ERROR: "validation-label-error",
+};
+
+function LimitedValidationLabel(props: ValidationLabelProps): JSX.Element {
+    const className = `${labelTypeMap[props.type]} line-cut`;
```
Comment (JulianWielga) [inline designer/client/src/components/modals/ValidationLabels.tsx:62]: prefer removing hardcoded, not connected, global classnames

**Review by JulianWielga — CHANGES_REQUESTED** [review]

File: designer/client/src/components/modals/ValidationLabels.tsx [hunk 8]
```diff
@@ -29,22 +27,42 @@ export default function ValidationLabels(props: Props) {
...
+    return (
+        <span className={className} title={props.tooltipMessage}>
```
Comment (JulianWielga) [inline designer/client/src/components/modals/ValidationLabels.tsx:64]: at now title is ok
should be changed to sth "better" later

File: designer/client/src/stylesheets/processes.styl (hunk 1 above)
Comment (mslabek) [reply designer/client/src/stylesheets/processes.styl]: done

File: designer/client/src/components/graph/node-modal/FieldLabel.tsx (hunk 2 above)
Comment (mslabek) [reply designer/client/src/components/graph/node-modal/FieldLabel.tsx]: done

File: designer/client/src/components/graph/node-modal/FieldLabel.tsx (hunk 3 above)
Comment (mslabek) [reply designer/client/src/components/graph/node-modal/FieldLabel.tsx]: right, unnecessary abstraction, removed

File: designer/client/src/components/modals/ValidationLabels.tsx (hunk 4 above)
Comment (mslabek) [reply designer/client/src/components/modals/ValidationLabels.tsx]: done

File: designer/client/src/components/modals/ValidationLabels.tsx (hunk 6 above)
Comment (mslabek) [reply designer/client/src/components/modals/ValidationLabels.tsx]: right, I changed this map to a switch - ts compiler now checks this

File: designer/client/src/components/modals/ValidationLabels.tsx (hunk 7 above)
Comment (mslabek) [reply designer/client/src/components/modals/ValidationLabels.tsx]: done

**Review by JulianWielga — CHANGES_REQUESTED** [review]
this change is in a different project

**Review by JulianWielga — CHANGES_REQUESTED** [review]
this also

**Review by JulianWielga — APPROVED** [review]

**Pull request merged**